Earthfile

```diff
@@ -18,20 +18,24 @@ copy-src:
     FROM +build-deps
     COPY --dir Cargo.toml Cargo.lock deny.toml wolfssl wolfssl-sys ./
 
+# build-dev builds with the Cargo dev profile and produces debug artifacts
 build-dev:
     FROM +copy-src
     DO rust-udc+CARGO --args="build"
     SAVE ARTIFACT target/debug /debug AS LOCAL artifacts/debug
 
+# build-release builds with the Cargo release profile and produces release artifacts
 build-release:
     FROM +copy-src
     DO rust-udc+CARGO --args="build --release"
     SAVE ARTIFACT target/release /release AS LOCAL artifacts/release
 
+# run-tests executes all unit and integration tests via Cargo
```

Comment:
`via Cargo` may not be needed since all the targets are via cargo

Reply:
I think `via Cargo` is useful because we can make this information available to any user of `earthly doc` without needing the user dig into the `Earthfile` itself. 

I did wonder whether it was too much detail to specify `via Cargo`, but I kept it because:
1) https://docs.earthly.dev/docs/earthly-command#earthly-doc examples also have the same amount of detail about `go build`
2) Comments that are out of sync with the implementation are confusing, but I doubt the comment will easily get out of sync with the implementation.  That's because the `DO rust-udc+CARGO --args="test"` is a few lines after.  If we change the implementation to now use Cargo anymore, I'm sure it will be obvious to update that comment.
3) I see from https://doc.rust-lang.org/cargo/commands/cargo-test.html that Cargo picks a bunch of defaults for you when running tests, so I think letting the user know that it's `via Cargo` is important, because that clues them in that the behaviour of the command will be according to Cargo's defaults.

Reply:
Yeah sounds good. 
Can we please add about `cargo-llvm` and `cargo deny` in their respective targets to be uniform ?

Reply:
Sure thing, updated the PR!